backend/src/main/java/com/sscatalog/specialistsservicescatalog/services/SpecialistsService.java

```diff
@@ -36,11 +43,44 @@ public SpecialistsService(SpecialistRepository specialistRepository,
         this.serviceRequestRepository = serviceRequestRepository;
     }
 
-    public void becomeSpecialist(User user) {
-        Specialist specialist = new Specialist();
+    public String becomeSpecialist(User user, BecomeSpecialistRequest request) {
+        Account account;
+        AccountLink accountLink;
+        try {
+            account = createStripeAccount();
+            accountLink = generateAccountLink(account, request.getRefreshUrl(), request.getReturnUrl());
+        } catch (StripeException exception) {
+            throw new ApiException(HttpStatus.INTERNAL_SERVER_ERROR, "Creating Stripe account failed", exception);
+        }
+
+        Specialist specialist = new Specialist(account.getId());
         specialist.setUser(user);
 
         specialistRepository.save(specialist);
```

Comment:
Why controllers, services etc. are named in plural ('UsersController', 'SpecialistsController', 'SpecialistsService') but their repositories are named in single ?